docs/google/best_practices.md

```diff
@@ -66,16 +66,16 @@ qubits = cirq.GridQubit.rect(2, 5)
 for i in range(100):
     my_circuit.append(cirq.H.on_each(*qubits))
     for q in qubits:
-        my_circuit.append(cirq.measure(q, key=f'm{q}'))
+        my_circuit.append(cirq.measure(q, key=cirq.MeasurementKey.parse_serialized(f'{i}:m{q}')))
 
 # The same circuit, but defined using CircuitOperations.
 # This is ~1000x smaller when serialized!
-sub_circuit = cirq.Circuit(cirq.H(qubits[0]), cirq.measure(qubits[0], key='m'))
-circuit_op = cirq.CircuitOperation(sub_circuit.freeze())
-circuit_op = circuit_op.with_qubits([q])
-circuit_op = circuit_op.with_measurement_key_mapping({'m': f'm{q}'})
-circuit_op = circuit_op.repeat(100)
-short_circuit = cirq.Circuit(circuit_op for q in qubits)
+q = cirq.NamedQubit("q")
```

Comment:
This `q` should be renamed to avoid collision with `for q in qubits`.

Reply:
done